### PR TITLE
Orderのメソッドを実装

### PR DESCRIPTION
**Pull request by tkeshun**
Orderのメソッドのロジックを修正

### CODE REVIEW

File: src/main/java/com/example/domain/Order.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
  *
  */
 public class Order {
-
+	private final double TAX_RATE = 0.1;
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'TAX_RATE' must contain no more than '1' consecutive capital letters.

Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MemberNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Member name 'TAX_RATE' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'.

File: src/main/java/com/example/domain/Order.java [hunk 2]
```diff
@@ -162,4 +162,33 @@ public String toString() {
 				+ ", paymentMethod=" + paymentMethod + "]";
 	}
 
+	/**
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:165]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/domain/Order.java [hunk 3]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+	 * 注文商品の合計金額の消費税額を返すメソッド.
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:166]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/domain/Order.java [hunk 4]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+	 *
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:167]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/domain/Order.java [hunk 5]
```diff
@@ -162,4 +162,33 @@ public String toString() {
 				+ ", paymentMethod=" + paymentMethod + "]";
 	}
 
+	/**
+	 * 注文商品の合計金額の消費税額を返すメソッド.
+	 * 
+	 * @return 消費税分の金額
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:168]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@return' should be preceded with an empty line.

Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:168]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/domain/Order.java [hunk 6]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+	 */
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:169]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/domain/Order.java [hunk 7]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+	public int getTax() {
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:170]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:170]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/domain/Order.java [hunk 8]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+		int basePrice = 0;
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:171]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:171]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/domain/Order.java [hunk 9]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+		// 商品ごとの合計金額を合算
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:172]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/domain/Order.java [hunk 10]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+		for (var orderItem : this.orderItemList) {
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:173]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:173]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'for' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/domain/Order.java [hunk 11]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+			basePrice += orderItem.getSubTotal();
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:174]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/domain/Order.java [hunk 12]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+		}
+		// 小数点以下切り捨て
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:176]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/domain/Order.java [hunk 13]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+		int tax = (int) (basePrice * TAX_RATE);
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:177]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/domain/Order.java [hunk 14]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+		return tax;
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:178]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:178]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/domain/Order.java [hunk 15]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+	}
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:179]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:179]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/domain/Order.java [hunk 16]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+
+	/**
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:181]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/domain/Order.java [hunk 17]
```diff
@@ -162,4 +162,33 @@ public String toString() {
...
+	 * 注文の合計を返すメソッド.
```
Comment (tkeshun) [inline src/main/java/com/example/domain/Order.java:182]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

**Review by tkeshun — COMMENTED** [review]
Remaining comments which cannot be posted as a review comment to avoid GitHub Rate Limit

<details>
<summary>checkstyle</summary>

[src/main/java/com/example/domain/Order.java|183 col 1|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L183) Line contains a tab character.
[src/main/java/com/example/domain/Order.java|184|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L184) Javadoc tag '@return' should be preceded with an empty line.
[src/main/java/com/example/domain/Order.java|184 col 1|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L184) Line contains a tab character.
[src/main/java/com/example/domain/Order.java|185 col 1|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L185) Line contains a tab character.
[src/main/java/com/example/domain/Order.java|186 col 1|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L186) Line contains a tab character.
[src/main/java/com/example/domain/Order.java|186 col 9|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L186) 'method def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/com/example/domain/Order.java|187 col 1|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L187) Line contains a tab character.
[src/main/java/com/example/domain/Order.java|187 col 17|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L187) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/domain/Order.java|188 col 1|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L188) Line contains a tab character.
[src/main/java/com/example/domain/Order.java|189 col 1|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L189) Line contains a tab character.
[src/main/java/com/example/domain/Order.java|189 col 17|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L189) 'for' has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/domain/Order.java|190 col 1|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L190) Line contains a tab character.
[src/main/java/com/example/domain/Order.java|190 col 25|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L190) 'for' child has incorrect indentation level 24, expected level should be 6.
[src/main/java/com/example/domain/Order.java|191 col 1|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L191) Line contains a tab character.
[src/main/java/com/example/domain/Order.java|191 col 17|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L191) 'for rcurly' has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/domain/Order.java|192 col 1|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L192) Line contains a tab character.
[src/main/java/com/example/domain/Order.java|192 col 17|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L192) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/domain/Order.java|193 col 1|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L193) Line contains a tab character.
[src/main/java/com/example/domain/Order.java|193 col 9|](http://github.com/ecsite-project/ec-202304c/blob/b1360431504cfff8065ad335a6c2a1b82badb1ab/src/main/java/com/example/domain/Order.java#L193) 'method def rcurly' has incorrect indentation level 8, expected level should be 2.
</details>

**Review by uedamiku — APPROVED** [review]

**Pull request merged**